bcl2fastq/index_summary.py

```diff
@@ -187,6 +237,15 @@ def update_unknown_index_pie(run_json):
      dep.Input('pruned_unknown_hidden', 'children')]
 )
 def update_pie_chart(run_alias, known_json, unknown_json):
+    """ When inputs are changed function os rerun to get updated pie chart
+            Attributes:
+                known(list): json format of known indices
+                pruned(list): json format of unknown indices
```

Comment:
For `known` and `pruned`: probably `str` type, and can remove the `json format of`